assets/js/actions/index.js

```diff
@@ -0,0 +1,10 @@
+import {
```

Comment:
It's important that you create separate files for each type of action (like, `battle-details.js` contains all actions related to the battle details), so you don't mix unrelated actions. Then, import/export the actions on `index.js`